### PR TITLE
Adding Missing Holiday for Barcelona and Catalonia

### PR DESCRIPTION
**Pull request by kkawakam**

### CODE REVIEW

**Comment by svendiedrichsen** [comment]
@kkawakam I left some comments. Besides this thank you very much.

**Comment by kkawakam** [comment]
@svendiedrichsen I don't see any comments

**Comment by svendiedrichsen** [comment]
@kkawakam I mean review comments with the source code changes.

File: src/main/resources/holidays/Holidays_es.xml [hunk 1]
```diff
@@ -77,8 +77,14 @@
   		<tns:Fixed month="JUNE" day="24" descriptionPropertiesKey="ST_JOHN"/>
   		<tns:Fixed month="SEPTEMBER" day="11" descriptionPropertiesKey="REGIONAL"/>
 		<tns:Fixed month="AUGUST" day="15" descriptionPropertiesKey="ASSUMPTION_DAY"/>
+		<tns:Fixed month="DECEMBER" day="26" descriptionPropertiesKey="BOXING_DAY"/>
   		<tns:ChristianHoliday type="EASTER_MONDAY"/>
   	</tns:Holidays>
+	<tns:SubConfigurations hierarchy="b" description="Barcelona">
```
[the remark this answers is not in the document]
Comment (kkawakam) [reply src/main/resources/holidays/Holidays_es.xml]: Ah, I thought these were supposed to match the ISO codes (https://en.wikipedia.org/wiki/ISO_3166-2:ES)? I will use `bcn` instead  if that is okay with you

[the remark this answers is not in the document]
Comment (svendiedrichsen) [reply src/main/resources/holidays/Holidays_es.xml]: I'm fine with that too.

**Pull request merged**